cli/cmd/clean.go

```diff
@@ -0,0 +1,154 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/running"
+)
+
+var forceRemove bool
+
+// NewCleanCmd creates clean command.
+func NewCleanCmd() *cobra.Command {
+	var cleanCmd = &cobra.Command{
+		Use:   "clean [INSTANCE_NAME]",
+		Short: "Clean instance(s) files",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := modules.RunCmd(&cmdCtx, cmd.Name(), &modulesInfo, internalCleanModule, args)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+		},
+	}
+
+	cleanCmd.Flags().BoolVarP(&forceRemove, "force", "f", false, "do not ask for confirmation")
+
+	return cleanCmd
+}
+
+func collectFiles(list *[]string, dirname string) error {
+	err := filepath.Walk(dirname,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if !info.IsDir() {
+				*list = append(*list, path)
+			}
+			return nil
+		})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func askConfirm(question string) (bool, error) {
```

Comment:
It could be useful to have "ask for confirmation" functionality in common place like utils or cli-utils, so other command can use it.

Reply:
moved to utils.go